tests/BUILD

```diff
@@ -14,6 +14,9 @@ appimage_test(
     size = "small",
     args = ["--appimage-extract-and-run"],  # One way to run if no libfuse2 is available
     binary = ":test_sh",
+    target_compatible_with = [
```

Comment:
Makes sense! I wonder if this should be added automatically for all AppImage and appimage_test rules. Might have to wrap them in macros for that though. WDYT? Maybe that would be a good follow up PR.

Nit: single item lists on single line please :)
Same for the other occurrences.

Reply:
Let's leave this for a follow up PR.